src/lib/src/ng-select.component.spec.ts

```diff
@@ -145,6 +118,86 @@ describe('NgSelectComponent', function () {
         }));
     });
 
+    describe('Pre-selected model', () => {
+        it('should select by valueKey when primitive type', fakeAsync(() => {
```

Comment:
Wrap these 3 tests in `describe('single')` because where is block `describe('multiple')`